.github/ISSUE_TEMPLATE/3-help.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "⁉️ Need help with Apollo Voyager Server?"
+about: Please file an issue in our repo.
+
+---
```

Comment:
Is this template appears for who click on in the button? Is GitHub able to work with more than one issue template?